spec/views/spree/layouts/admin/_login_nav.html.haml_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+describe 'spree/layouts/admin/_login_nav.html.haml' do
+
+  it 'display the application version in admin navbar' do
+    def  view.spree_current_user
+      FactoryGirl.build(:user)
+    end
```

Comment:
Is this common practice in the Rails community? rails-rspec seems to [suggest](https://github.com/rspec/rspec-rails#view-specs) using `assign`.